### ISSUE TITLE
The item's actions card is not accessible enough through voiceover

### ISSUE THREAD

**Issue by pablocarmu** [owncloud/ios-app#264]
To present the actions that the user could apply over an item we use a _card_ with the actions inside. This card is easy to use and provides an interactive and natural way of presenting those options.

But the thing changes when you try to use this screen with Apple's _voice over_ accessibility mechanism enabled. The regular accessibility 3-finger-gesture does not work so dismiss this screen is really hard if you don't want to make any action.
I've tried to set the background dimming view as an accessibility element but, give it some accessibility hints but it does not work fine.

I've tried the Apple's own implementation of this _cards_ in the Maps app and the experience was as bad as our implementation, almost impossible to drag if you have _voice over_ enabled, only force-touch in the card seems to make the drag easier, but not always.

An example of our current card view for item's actions:

[<img src="https://user-images.githubusercontent.com/8929738/52467738-d9ba7b80-2b86-11e9-9d73-e13db69e2cc1.jpeg" width="400px"/>](https://user-images.githubusercontent.com/8929738/52467738-d9ba7b80-2b86-11e9-9d73-e13db69e2cc1.jpeg) | ![ownCloud_none](https://user-images.githubusercontent.com/214010/81505766-fb5a6380-92f1-11ea-8a2d-11fe227f7787.gif)
-- | --

cc/ @felix-schwarz @jesmrec @hosy @mneuwert @javiergonzper @michaelstingl

**Comment by michaelstingl** [comment]
@pablocarmu thanks for checking!

**Comment by michaelstingl** [comment]
Maps | Podcasts | Photo (Edit)
-- | -- | --
![Maps_X+Bar](https://user-images.githubusercontent.com/214010/81505788-26dd4e00-92f2-11ea-976a-c8f2f1b30e3a.gif) | ![Podcast_Bar](https://user-images.githubusercontent.com/214010/81505794-3361a680-92f2-11ea-954a-b728d2ce552c.gif) | ![Photos_edit_X+Space](https://user-images.githubusercontent.com/214010/81505799-38bef100-92f2-11ea-8d60-accb85ef8d5f.gif)
X-button & dragging-bar | dragging-bar | X-button & space above card

**Comment by mneuwert** [comment]
@michaelstingl I think it is pretty easy to fix.. would be like 3 lines fix for current solution: 
- Add accessibility label to the drag handle
- Declare it as accessibility element
- Make a tap target a bit bigger by augmenting accessibilityFrame property

And we can consider adding additional X button of course

**Comment by michaelstingl** [comment]
> And we can consider adding additional X button of course

We built custom UI back in 2018. Anything more native from Apple we could use instead?

**Comment by felix-schwarz** [comment]
@michaelstingl With iOS 13, Apple almost got us there - with the way view controllers are presented by default. But although that presentation looks similar to cards, it's not possible to define a different top "stop" point, so when using that presentation style, they always fill almost the entire screen. Hopefully, we'll get a proper card UI component in iOS 14.

**Comment by michaelstingl** [comment]
Okay, then we should go with @mneuwert simple approach for now.

**Comment by jesmrec** [comment]
Done!
